### ISSUE TITLE
non-writable files in supposedly writable dir breaks install 1.2.8 -> 1.2.11

### ISSUE THREAD

**Issue by vKnmnn** [Cacti/cacti#3508]
Cacti on Ubuntu Server 18, php 7.2

last week i upgraded to cacti 1.2.11 from 1.2.8

all checks were ok, but the installation halted at about 40 percent. i left it running for a few hours and came back to cacti supposedly in version 1.2.11 but with a lot fewer devices.

The installation log didn't show in my browser, but i saw on the commandline errors popping up about permission errors on some files.
```
20.04.2020 14:06:10 - IMPORT Writing file: /usr/share/cacti/site/scripts/ss_poller.php
20.04.2020 14:06:10 - IMPORT Write Status file: /usr/share/cacti/site/scripts/ss_poller.php, with Status not writable
20.04.2020 14:06:10 - IMPORT Writing file: /usr/share/cacti/site/scripts/cacti_user_stats.php
20.04.2020 14:06:10 - IMPORT Write Status file: /usr/share/cacti/site/scripts/cacti_user_stats.php, with Status not writable
20.04.2020 14:06:10 - IMPORT Writing file: /usr/share/cacti/site/resource/script_server/cpoller.xml
20.04.2020 14:06:10 - IMPORT Write Status file: /usr/share/cacti/site/resource/script_server/cpoller.xml, with Status not writable
20.04.2020 14:06:10 - IMPORT Writing file: /usr/share/cacti/site/resource/script_server/webseer.xml
20.04.2020 14:06:10 - IMPORT Write Status file: /usr/share/cacti/site/resource/script_server/webseer.xml, with Status not writable
20.04.2020 14:06:10 - IMPORT Writing file: /usr/share/cacti/site/resource/script_server/gexport.xml
20.04.2020 14:06:10 - IMPORT Write Status file: /usr/share/cacti/site/resource/script_server/gexport.xml, with Status not writable
```

the devices are still in the database, but cacti doesn't list them in the "management -> devices" list. I am able to inspect the devices by putting their URL into the address bar in edit.php, however.
So now i could use some help getting those devices back into the list.

Thanks for your help

**Comment by netniV** [comment]
This is likely directly related to #3438.  The status failures are interesting as I don't believe anyone else has reported those nor left their install running long enough to see those show up.  Hopefully, that has been resolved and your devices are there and will show up, you just also need the fix from #3424 which unfortunately hid them from you.

If that's the case, I can close this off as a duplicate.

**Comment by vKnmnn** [comment]
patch from #3424 fixes the missing hosts.
I'm looking forward do 1.2.12 to see how the installation goes.
